services/keybase/keybase-btc.service.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const KeybaseProfile = require('./keybase-profile')
+const Joi = require('joi')
+
+const bitcoinAddressFoundSchema = Joi.object({
+  them: Joi.array()
+    .items(
+      Joi.object({
+        cryptocurrency_addresses: Joi.object({
+          bitcoin: Joi.array()
+            .items(
+              Joi.object({
+                address: Joi.string().required(),
+              }).required()
+            )
+            .required(),
+        }).required(),
+      }).required()
+    )
+    .min(0)
+    .max(1),
+}).required()
+
+const profileNotFoundSchema = Joi.object({
+  them: Joi.array().empty(),
+}).required()
+
+const bitcoinAddressFoundOrNotFound = Joi.alternatives(
+  bitcoinAddressFoundSchema,
+  profileNotFoundSchema
+)
+
+module.exports = class KeybaseBTC extends KeybaseProfile {
+  static get apiVersion() {
+    return '1.0'
+  }
+
+  static get category() {
+    return 'social'
+  }
+
+  static get route() {
+    return {
+      base: 'keybase/btc',
+      pattern: ':username',
+    }
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'btc',
+      color: 'informational',
+    }
+  }
+
+  async handle({ username }) {
+    const options = {
+      form: {
+        usernames: username,
+        fields: 'cryptocurrency_addresses',
+      },
+    }
+
+    const data = await this.fetch({
+      schema: bitcoinAddressFoundOrNotFound,
+      options,
+    })
+
+    if (data.them.length === 0) {
+      return {
+        message: 'profile not found',
+        color: 'critical',
+      }
+    }
+
+    const bitcoinAddresses = data.them[0].cryptocurrency_addresses.bitcoin
+
+    if (bitcoinAddresses.length === 0) {
+      return {
+        message: 'no bitcoin addresses found',
+        color: 'inactive',
+      }
+    }
+
+    return this.constructor.render({ address: bitcoinAddresses[0].address })
+  }
+
+  static render({ address }) {
+    return {
+      message: address,
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Keybase BTC',
+        namedParams: { username: 'Keybase username' },
```

Comment:
For the example params, we want to list a real value so that a user could plug that in and get a real badge. Let's put a real username here (I'd suggest putting your `skyplabs` if you don't mind 😄)

Reply:
Ahah done :+1: